src/managementgroup/README.md

```diff
@@ -0,0 +1,109 @@
+# Azure CLI managementgroup Extension #
+This is the extension for managementgroup
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name managementgroup
+```
+
+### Included Features ###
+#### managementgroup management-group ####
+##### Create #####
+```
+az managementgroup management-group create --display-name "ChildGroup" \
+    --id "/providers/Microsoft.Management/managementGroups/ParentGroup" --group-id "ChildGroup" 
+```
+##### Show #####
+```
+az managementgroup management-group show --group-id "20000000-0001-0000-0000-000000000000"
+```
+##### Show #####
+```
+az managementgroup management-group show --expand "ancestors" --group-id "20000000-0001-0000-0000-00000000000"
+```
+##### Show #####
+```
+az managementgroup management-group show --expand "children" --group-id "20000000-0001-0000-0000-000000000000"
+```
+##### Show #####
+```
+az managementgroup management-group show --expand "path" --group-id "20000000-0001-0000-0000-000000000000"
+```
+##### Show #####
+```
+az managementgroup management-group show --expand "children" --recurse true \
+    --group-id "20000000-0001-0000-0000-000000000000" 
+```
```

Comment:
> ##### Show #####

Could we have a more detailed description of this title, or could we put similar contents together under one title